.config/tsaoptions.json

```diff
@@ -0,0 +1,15 @@
+{
+  "instanceUrl"        :  "https://msazure.visualstudio.com",
+  "areaPath"           :  "One\\TelReachSDKTeam",
+  "projectName"        :  "One",
+  "iterationPath"      :  "One",
+  "notificationAliases":  [
+    "Trask.Stalnaker@microsoft.com",
+    "Helen.Yang@microsoft.com"
+  ],
+  "codebaseAdmins"     : [
+    "Trask.Stalnaker@microsoft.com",
```

Comment:
same here.

Reply:
the doc for codebaseAdmins doesn't appear to be email addresses? https://onebranch.visualstudio.com/OneBranch/_wiki/wikis/OneBranch.wiki/4923/SDL-Bugfiling-(TSA)?anchor=codebase-admins

Reply:
removed.. realized that i don't need to change it.. whatever the default will be fine.